.circleci/config.yml

```diff
@@ -335,7 +335,32 @@ jobs:
           registry: registry.heroku.com
           tag: latest
       - run: heroku container:release -a simple-hub-staging simple-hub
-
+      
+  deploy-persistent-seeder:
+    working_directory: /home/circleci/project
+    resource_class: large
+    docker:
+      - image: circleci/node:10.16.3-browsers
+    steps:
+      - setup_remote_docker
+      - checkout
+      - <<: *restore_dep
+      - <<: *restore_lib
+      - run: rm -rf ./patches && cd ./packages/ && find . -maxdepth 1 ! -path "./channel-provider" ! -path "./client-api-schema" ! -path "./e2e-tests" ! -path "./devtools" -exec rm -rf {} ';'
+      - run: yarn --frozen-lockfile
+      - run: curl https://cli-assets.heroku.com/install.sh | sh
+      - run: heroku container:login
+      - docker/build:
+          cache_from: registry.heroku.com/persistent-seeder-staging/persistent-seeder:latest
+          dockerfile: packages/e2e-tests/persistent-seeder-deployment/persistent-seeder.dockerfile
```

Comment:
As a general comment, it seems odd to me that `persistent-seeder` is defined inside of `packages/e2e-tests`.

Reply:
The reason for that is that it has the same dependencies as, and leans on helper functions defined in that package. It is functionally extremely similar to our e2e-tests (only it does not run as an actual test). Alternatives include:
- moving to an entirely new package
- moving under web3torrent, but that involves either
  - making web3torrent depend on e2e-tests 🤮 
  - duplicating the e2e-tests code 🤮 

I'm not sure it is worth it at this point.